spk/bazarr/src/dsm-control.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Package
+PACKAGE="bazarr"
+DNAME="Bazarr"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python3"
+BAZARR_DIR="${INSTALL_DIR}/share/${PACKAGE}"
+VAR_DIR="${INSTALL_DIR}/var";
+
+export PATH="${INSTALL_DIR}/bin:${PYTHON_DIR}/bin:${PATH}"
+export LC_ALL=en_US.utf8
```

Comment:
This was made to enforce default locale settings as it seems to not really be handled in Synology mechanism to bootstrap package services. Unless someone knows how to read it... Personally I would rather go for the /etc/profile but then... It would introduce a heavy dependency here I believe so I went for the "shortest": default settings which can be overridden (along with other things) by defining a profile script under the var directory.

Reply:
I like the solution you came up with here. Default to `en_US.UTF-8` but one can easily override and/or provide additional environment variables via the optional `profile` file. 👍